### ISSUE TITLE
torch.sparse has no attribute softmax

### ISSUE THREAD

**Issue by InstantWindy** [pytorch/pytorch#82715]
pytorch version : 1.4.0 

torch.sparse has no attribute softmax
